### ISSUE TITLE
Train and Val set for target domain same?

### ISSUE THREAD

**Issue by AadSah** [chenchao666/HoMM-Master#1]
Hi @chenchao666 

Thanks for the great work. It seems that the text file used for generating the training and validation sets for the target domain are the same in the code, for the Office-31 dataset. Namely, it is webcam.txt in the finetune.py file. Can you please clarify this?

**Comment by chenchao666** [comment]
> Hi @chenchao666
> 
> Thanks for the great work. It seems that the text file used for generating the training and validation sets for the target domain are the same in the code, for the Office-31 dataset. Namely, it is webcam.txt in the finetune.py file. Can you please clarify this?

Yeap， this is a common setting in unsupervised domain adaptation. Since the target domain has no labels, the unsupervised domain adaptation aims to train a classifier with the labeled source samples and unlabeled target samples. Therefore, the target domain samples are used for both traing and test. Just same to the unsupervised learning.

**Comment by AadSah** [comment]
> > Hi @chenchao666
> > Thanks for the great work. It seems that the text file used for generating the training and validation sets for the target domain are the same in the code, for the Office-31 dataset. Namely, it is webcam.txt in the finetune.py file. Can you please clarify this?
> 
> Yeap， this is a common setting in unsupervised domain adaptation. Since the target domain has no labels, the unsupervised domain adaptation aims to train a classifier with the labeled source samples and unlabeled target samples. Therefore, the target domain samples are used for both traing and test. Just same to the unsupervised learning.

Oh. Got it. Thanks a lot. Actually I was having a look at some common datasets used for unsupervised domain adaptation. The DomainNet dataset seems to have exclusive sets for train and test. So, thought if Office-31 had. 
          Anyways, thanks. And can you please provide the validation split, the paper used for the other two domains of the Office-31 dataset namely 'amazon' and 'dslr'? The repository seems to have only the validation split for the 'webcam' domain only.

**Comment by chenchao666** [comment]
> > > Hi @chenchao666
> > > Thanks for the great work. It seems that the text file used for generating the training and validation sets for the target domain are the same in the code, for the Office-31 dataset. Namely, it is webcam.txt in the finetune.py file. Can you please clarify this?
> > 
> > 
> > Yeap， this is a common setting in unsupervised domain adaptation. Since the target domain has no labels, the unsupervised domain adaptation aims to train a classifier with the labeled source samples and unlabeled target samples. Therefore, the target domain samples are used for both traing and test. Just same to the unsupervised learning.
> 
> Oh. Got it. Thanks a lot. Actually I was having a look at some common datasets used for unsupervised domain adaptation. The DomainNet dataset seems to have exclusive sets for train and test. So, thought if Office-31 had.
> Anyways, thanks. And can you please provide the validation split, the paper used for the other two domains of the Office-31 dataset namely 'amazon' and 'dslr'? The repository seems to have only the validation split for the 'webcam' domain only.

all the data has been included, you only need to modifiy the name of domain 'webcam' 'dslr' 'amazon'

**Comment by AadSah** [comment]
> > > > Hi @chenchao666
> > > > Thanks for the great work. It seems that the text file used for generating the training and validation sets for the target domain are the same in the code, for the Office-31 dataset. Namely, it is webcam.txt in the finetune.py file. Can you please clarify this?
> > > 
> > > 
> > > Yeap， this is a common setting in unsupervised domain adaptation. Since the target domain has no labels, the unsupervised domain adaptation aims to train a classifier with the labeled source samples and unlabeled target samples. Therefore, the target domain samples are used for both traing and test. Just same to the unsupervised learning.
> > 
> > 
> > Oh. Got it. Thanks a lot. Actually I was having a look at some common datasets used for unsupervised domain adaptation. The DomainNet dataset seems to have exclusive sets for train and test. So, thought if Office-31 had.
> > Anyways, thanks. And can you please provide the validation split, the paper used for the other two domains of the Office-31 dataset namely 'amazon' and 'dslr'? The repository seems to have only the validation split for the 'webcam' domain only.
> 
> all the data has been included, you only need to modifiy the name of domain 'webcam' 'dslr' 'amazon'

Great. Thanks.
